### ISSUE TITLE
Questions on CLIP calibration

### ISSUE THREAD

**Issue by DianCh** [google-research/robustness_metrics#32]
Hi, I was wondering how you calculated the confidence for CLIP models with their zero-shot predictions. Specifically:

- Did you take the softmax of the `logits` calculated by the following snippet (taken from the original CLIP paper)?
- If so, what temperature value `t` did you use in your "unscaled" setting and how did you adjust the temperature in your "scaled" setting?
- If not, could you help clarify what values are used as confidence?

<img width="525" alt="Screen Shot 2021-07-17 at 11 35 19 AM" src="https://user-images.githubusercontent.com/32043460/126046554-7e4d9784-e7d6-48b4-97c9-855b07ffd8de.png">

Thank you for the wonderful work of the paper!

**Comment by josipd** [comment]
Hello,

our instantiation of the CLIP model is checked in here: https://github.com/google-research/robustness_metrics/blob/5a380c4ec11fa85255b5db643c64efa42d749110/robustness_metrics/models/clip.py
